libs/cardano-data/src/Data/CanonicalMaps.hs

```diff
@@ -36,14 +36,14 @@ instance CanonicalZero Integer where
   zeroC = 0
   joinC = (+)
 
-instance (Ord k, CanonicalZero v) => CanonicalZero (Map k v) where
+instance (Show k, Show v, Ord k, CanonicalZero v) => CanonicalZero (Map k v) where
   zeroC = Map.empty
   joinC = canonicalMapUnion joinC
 
 -- Note that the class CanonicalZero and the function canonicalMapUnion are mutually recursive.
 
 canonicalMapUnion ::
-  (Ord k, CanonicalZero a) =>
+  (Show k, Show a, Ord k, CanonicalZero a) =>
```

Comment:
What's with the `Show` constraints?

Reply:
Leftover from debug tracing, that would be my guess.